TYPO3.TYPO3CR/Classes/TYPO3/TYPO3CR/Eel/FlowQueryOperations/FilterOperation.php

```diff
@@ -129,7 +129,17 @@ protected function evaluateOperator($value, $operator, $operand) {
 				$isOfType = $value->getNodeType()->isOfType($operand[0] === '!' ? substr($operand, 1) : $operand);
 				return $operand[0] === '!' ? $isOfType === FALSE : $isOfType;
 			}
+		} else if ($operator === '!instanceof' && $value instanceof \TYPO3\TYPO3CR\Domain\Model\NodeInterface) {
```

Comment:
thanks a lot for your comment and input! 
your approach is indeed less prone to errors and adjustments to the `instanceof` operation would automatically apply to `!instanceof`. So full ack! Will apply the change as soon as i have access to my dev machine.